charter.md

```diff
@@ -59,6 +59,7 @@ The SPICE WG's work items will not require nor forbid the use of JSON-LD. Retain
 - Availability (efficiently information processing, minimizing data in flight, reducing carbon cost for storage and transmission, and supporting offline / paper credential use cases)
 - Semantic interchangeability (ensuring terms are understood and used consistently, in a global or local context)
 - Usability and Feasibility ( The core operations must be easy to understand and use, and possible to integrate with existing platforms)
+- Accountability (what format and protocol considerations can help mitigates abusive or excessive requests for credential presentation?  what mitigates abuse by issuers?)
```

Comment:
I like this, and as one example, we might consider providing guidance on risks associated with requiring or forbidding attestation.